### DISCUSSION TITLE
find too many unsuccessfully terminated jobs

### DISCUSSION THREAD

**Discussion by mayankaditya** [deepmodeling/dpgen#391] [Q&A]
Hi, I am getting the following error in the second iteration:

 **find too many unsuccessfully terminated jobs**

How to go ahead?

Thanks,
Mayank

**Comment by njzjz** [comment]
set `ratio_failed` in your json file to 1, i.e. 
```json
"ratio_failed": 1,
```

**Comment by mayankaditya** [comment]
There are two json files, and each of them has many sections where should I put this line.

Thanks,
Mayank

**Comment by njzjz** [comment]
`PARAM` file.

**Reply by warriors1523** [reply]
I set ratio_failed in this position of my paramter.json file ，but it didn't seem to work.
![1655864764359](https://user-images.githubusercontent.com/92622311/174946577-d1ce9862-75a6-4439-8d9f-46290fcebfba.png)

Thanks,
Zhang

**Comment by HuangJiameng** [comment]
I'd like to know the parameter "fp_style" in your param.json. If it's cp2k, a new PR will fix this problem.

**Reply by warriors1523** [reply]
The parameter "fp_style" is "VASP" in my param.json. The problem I meet is that some lammps calculations failed because of atoms lost at early model deviation. Currently I can manually delete failed tasks and restart them to let dpgen continue to work. Are there parameters in DPGEN to control the highest acceptable fail rate of MD calculation to let dpgen continue to work? Someone had the same problem as me before, but the key words he mentioned didn't seem to work. 
The link is as follows:
https://github.com/deepmodeling/dpgen/discussions/550
https://github.com/deepmodeling/dpgen/discussions/763
